app/src/main/res/values/strings.xml

```diff
@@ -344,6 +344,14 @@ Please report any problems you encounter via Github.</string>
 
     <string name="use_legacy_hashing_summary">Force Syncthing to use legacy hashing package for compatibility purposes</string>
 
+    <string name="use_folder_observer_title">Use FolderObserver</string>
+
+    <string name="use_folder_observer_summary">Use wrapper\'s FolderObserver for realtime change detection. Syncthing core\'s fsWatcher will be disabled on all folders.</string>
```

Comment:
These terms (FolderObserver, fsWatcher) make a lot of sense if you read the code, yet the user has no fucking clue what these are and what these do, so the description should be more human.

Reply:
... so I submit a new suggestion.:)